app/src/main/kotlin/com/intive/picover/main/navigation/model/NavigationType.kt

```diff
@@ -6,15 +6,13 @@ import androidx.compose.material3.windowsizeclass.WindowWidthSizeClass
 enum class NavigationType {
 	NAVIGATION_BAR,
 	NAVIGATION_RAIL,
-	NAVIGATION_DRAWER,
 	;
 
 	companion object {
 		fun fromWindowSize(windowSize: WindowSizeClass) =
 			when (windowSize.widthSizeClass) {
-				WindowWidthSizeClass.Medium -> NAVIGATION_RAIL
-				WindowWidthSizeClass.Expanded -> NAVIGATION_DRAWER
-				else -> NAVIGATION_BAR
+				WindowWidthSizeClass.Compact -> NAVIGATION_BAR
+				else -> NAVIGATION_RAIL
```

Comment:
1. use if-else
2. Please also simplify names, there is no need to repeat "NAVIGATION_" in every enum value

Reply:
Done

Reply:
Please check where I posted a comment. I was referring to `NavigationType` class and `fromWindowSize` function.

Reply:
Fixed :D